server/channels/api4/cloud.go

```diff
@@ -65,10 +65,15 @@ func (api *API) InitCloud() {
 
 func ensureCloudInterface(c *Context, where string) bool {
 	cloud := c.App.Cloud()
+	disabled := c.App.Config().CloudSettings.Disable != nil && *c.App.Config().CloudSettings.Disable
```

Comment:
nit: could move this after the `cloud == nil` check, but not a big deal.

Reply:
:+1: for checking `disabled` first